app/controllers/idv/sessions_controller.rb

```diff
@@ -31,6 +36,19 @@ def update
 
     private
 
+    def validation_errors
+      @_validation_errors ||= []
```

Comment:
Is this pattern of using `_` a common thing in ruby / rails?  Should we be using it more?  Does it have actual meaning in ruby?